src/modules/app-info/app-info.effects.ts

```diff
@@ -27,4 +36,32 @@ export class AppInfoEffects {
         );
     }),
   );
+
+  @Effect()
+  loadConfigSuccessEffect$ = this.actions$.pipe(
+    ofType(appInfoActions.LOAD_CONFIG_SUCCESS),
+    switchMap(() => {
+      console.log('Config loaded successfully');
```

Comment:
remove console.log

Reply:
I think Ammar wanted this. I can't remember exactly why... but it definitely made sense at the time.